projects/pytorch/README.md

```diff
@@ -24,81 +24,34 @@ Run Docker:
 
 Change directory to `/fuzz`:
 
-    # cd /fuzz
+    $ cd /fuzz
```

Comment:
in other projects readme we use '#' in commands that are proposed to be run inside the docker container. maybe change back to '#'? here and later

Reply:
Done